src/cli/execution.js

```diff
@@ -15,7 +32,10 @@ const execution = () => {
     return cli.showHelp(1)
   }
 
-  return scanFromFile(cli.input[0], cli.flags)
```

Comment:
I don't think moving scanFromFile responsability to here is the best way to do it. What about a flag or a third argument to pass what the output should be?

Reply:
This is bad for 2 reasons:
- we can't reuse the options on cli.
- execution files becames a lot more complex.